docs/WritingTests.asciidoc

```diff
@@ -611,6 +611,19 @@ based on company conventions:
 WORKER_CLASS = planet-earth,continent-antarctica,location-my_station
 --------------------------------------------------------------------------------
 
+The load of the system can affect test execution and cause failures due to delays of command execution and https://en.wikipedia.org/wiki/Thrashing_(computer_science)[thrashing]. If the average over a period of 15 minutes exceeds the specified value the worker will not accept new jobs.
+
+[caption="File: "]
+.workers.ini
+[source,ini]
+--------------------------------------------------------------------------------
+[global]
+# Set to 0 to disable.
+#CRITICAL_LOAD_AVG_THRESHOLD = 40
+--------------------------------------------------------------------------------
+
+Note that the value can also be overridden by setting `OPENQA_LOAD_AVG_FILE` in the environment, such as in the systemd service.
```

Comment:
The expression "the value" makes it sound like the env variable would allow overriding the threshold value set via CRITICAL_LOAD_AVG_THRESHOLD as shown above. However, the env variable really replaces the "file" the load is read from. This is only meant for testing/development. So I'd remove this whole paragraph about the env variable from this user-facing documentation.

Reply:
So I just realized this was still open...

I don't have a super strong opinion on where this should go, but ideally we document these variables **somewhere**. If you have a suggestion where to put it, let me know, otherwise I'd prefer to go with this instead of not documenting them.

Reply:
As I wrote the only better way I could see is to create a separate documentation document explaining all environment variables and ensuring with an automatic static check that all environment variables are also listed in such document, whatever the name or location which is secondary

Reply:
Oh. Re-reading it now I get what you mean it's wrong. I only got the part about it being meant for testing before 😅